llvm/lib/CodeGen/GlobalISel/LegalizerHelper.cpp

```diff
@@ -3330,16 +3330,17 @@ LegalizerHelper::LegalizeResult LegalizerHelper::lowerLoad(GAnyLoad &LoadMI) {
   LLT MemTy = MMO.getMemoryType();
   MachineFunction &MF = MIRBuilder.getMF();
 
-  unsigned MemSizeInBits = MemTy.getSizeInBits();
-  unsigned MemStoreSizeInBits = 8 * MemTy.getSizeInBytes();
+  unsigned MinMemSizeInBits = MemTy.getSizeInBits().getKnownMinValue();
+  unsigned MinMemStoreSizeInBits =
```

Comment:
You might need to rewrite this as `MemTy.getSizeInBytes() * 8` to use TypeSize. I think I remember that the multiplier has to be on the right hand side for the operator overloading.

Reply:
Or use https://llvm.org/doxygen/classllvm_1_1details_1_1FixedOrScalableQuantity.html#aa5b3e822013fe51665e9bddc4874cd48